index.html

```diff
@@ -27,94 +33,102 @@ <h4 class="connect">LET'S CONNECT</h4>
+            <div class="info">
+                <h2 class="work-title">Tonic</h2>
+                <div class="sub-desc">
+                    <h5 class="brand">CANOPY</h5>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="stack">Back End Dev</p>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="year">2015</p>
+                </div>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javascript</li>
+                </ul>
+                <button type="button" class="detail-btn">See Project</button>    
+            </div>
         </section>
 
-        <section class="works">
-            <a class="snapshoots"><img src="./assets/Snapshoot4.png" alt="Snapshoot4"></a>
-            <h2 class="work-title">Multi-Post <br> Stories</h2>
-            <section class="sub-desc">
-                <h5 class="brand">CANOPY</h5>
-                <img src="./assets/Counter.png" alt="dot">
-                <p class="stack">Back End Dev</p>
-                <img src="./assets/Counter.png" alt="dot">
-                <p class="year">2015</p>
-            </section>
-            <p class="description">
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
-            </p>
-            <ul class="languages">
-                <li class="lang">html</li>
-                <li class="lang">css</li>
-                <li class="lang">javascript</li>
-            </ul>
-            <button type="button" class="detail-btn">See Project</button>
+        <section class="work right">
+            <a class="snapshoots"><img class="big-img" src="./assets/Snapshoot4.png" alt="Snapshoot4"></a>
+            <div class="info">
+                <h2 class="work-title">Multi-Post Stories</h2>
+                <div class="sub-desc">
+                    <h5 class="brand">CANOPY</h5>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="stack">Back End Dev</p>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="year">2015</p>
+                </div>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javascript</li>
+                </ul>
+                <button type="button" class="detail-btn">See Project</button>    
+            </div>
+    
         </section>
```

Comment:
- [ ] Kindly ensure you follow the desktop design as we have in the template. In the Figma template design, we have alternating image positions whereas, in your implementation, all the images are aligned to the left. Kindly fix this.

  Your Implementation | Figma Design
  :---:|:---:
  ![tugsem-images](https://user-images.githubusercontent.com/36542986/159880758-2799f149-ca51-4902-a528-d2587974848c.jpg) | ![tugsem-images-figma](https://user-images.githubusercontent.com/36542986/159880762-6adb4788-7e41-4d34-93a7-2ec5e139fcdd.jpg)